### PR TITLE
update ember, ember-data, ember-cli to v2.8

### PR DESCRIPTION
**Pull request by camskene**

### CODE REVIEW

**Comment by kellyselden** [comment]
please remove testem.json

File: package.json [hunk 1]
```diff
@@ -19,23 +19,27 @@
...
     "ember-cli-uglify": "^1.2.0",
-    "ember-data": "^2.1.0",
+    "ember-data": "^2.8.0",
     "ember-data-fixture-adapter": "1.13.0",
-    "ember-disable-proxy-controllers": "^1.0.0",
-    "ember-export-application-global": "^1.0.4"
+    "ember-export-application-global": "^1.0.5",
+    "ember-load-initializers": "^0.5.1",
+    "ember-resolver": "^2.0.3",
+    "ember-welcome-page": "^1.0.1",
```
Comment (kellyselden) [inline package.json]: this isn't needed

File: bower.json [hunk 2]
```diff
@@ -1,16 +1,7 @@
...
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.1.0",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.2.0",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.6",
-    "ember-qunit": "0.4.10",
-    "ember-qunit-notifications": "0.0.7",
-    "ember-resolver": "~0.1.18",
-    "jquery": "^1.11.3",
-    "loader.js": "ember-cli/loader.js#3.2.1",
-    "qunit": "~1.18.0",
-    "moment": "^2.14.1"
```
Comment (kellyselden) [inline bower.json:14]: how is this coming in now? is it not needed anymore?

**Review by kellyselden — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (camskene) [reply package.json]: Done in da6a4b48172f3e64d70b4288ee213e0569b1ed2a

File: bower.json (hunk 2 above)
Comment (camskene) [reply bower.json:14]: It doesn't look like moment is used anywhere in the project?

**Pull request closed without merging**